src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/android/BrokerHelper.cs

```diff
@@ -195,12 +195,17 @@ internal static void SetBrokerResult(Intent data, int resultCode)
             {
                 string accessToken = data.GetStringExtra(BrokerConstants.AccountAccessToken);
                 DateTimeOffset expiresOn = BrokerProxy.ConvertFromTimeT(data.GetLongExtra(BrokerConstants.AccountExpireDate, 0));
+                string accountAuthority = data.GetStringExtra(BrokerConstants.AccountAuthority);
+                string accountUserInfoTenantId = data.GetStringExtra(BrokerConstants.AccountUserInfoTenantId);
+
                 UserInfo userInfo = BrokerProxy.GetUserInfoFromBrokerResult(data.Extras);
                 resultEx = new AuthenticationResultEx
                 {
                     Result = new AuthenticationResult("Bearer", accessToken, expiresOn)
```

Comment:
cleaner fix would be to create a TokenResponse object and call GetResultEx(). broker should also be returning id token. that way you don't have to construct everything